src/test-summary-workspace/test-summary.service.ts

```diff
@@ -91,7 +111,8 @@ export class TestSummaryWorkspaceService {
     private readonly testQualificationWorkspaceService: TestQualificationWorkspaceService,
     @Inject(forwardRef(() => QASuppDataWorkspaceService))
     private readonly qaSuppDataService: QASuppDataWorkspaceService,
-  ) {}
+  ) {
+  }
 
   async getTestSummaryById(testSumId: string): Promise<TestSummaryDTO> {
     const result = await this.repository.getTestSummaryById(testSumId);
```

Comment:
This should be updated to accept a transaction and conditionally use it (it should use the `withTransaction` utility since it calls a custom repository method):
```suggestion
  async getTestSummaryById(testSumId: string, trx?: EntityManager): Promise<TestSummaryDTO> {
    const result = await withTransaction(this.repository, trx).getTestSummaryById(testSumId);
```